src/Cake.Issues.Markdownlint/LogFileFormat/MarkdownlintLogfileHelper.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace Cake.Issues.Markdownlint.LogFileFormat
+{
+    using System;
+    using System.Text.RegularExpressions;
+
+    /// <summary>
+    /// Helper class for Markdownlint log files.
+    /// </summary>
+    internal static class MarkdownlintLogFileHelper
+    {
+        /// <summary>
+        /// Reads the RegEx for a Mardown issue line depending its line format.
+        /// </summary>
+        /// <param name="markdownIssueLine">Markdown issue line.</param>
+        /// <returns>The RegEx pattern to parse the Markdown issue line.</returns>
+        public static Regex GetRegexForMarkdownIssueLine(string markdownIssueLine)
```

Comment:
We don't need a static helper class for this. You can move this as a private method to the `MarkdownlintCliLogFileFormat` class.